tests/integration/features/features_int_test.go

```diff
@@ -204,7 +202,7 @@ var _ = Describe("feature cleanup", func() {
 				Expect(featuresHandler.Apply()).To(Succeed())
 
 				// then
-				featureTracker, err := getFeatureTracker("always-working-feature", appNamespace)
+				featureTracker, err := fixtures.GetFeatureTracker("always-working-feature", appNamespace, envTestClient)
```

Comment:
Same as with `GetNamespace` - lets pass the client first.

Reply:
5686115